src/cooccur.c

```diff
@@ -112,7 +64,7 @@ void hashinsert(HASHREC **ht, char *w, long long id) {
         htmp = (HASHREC *) malloc(sizeof(HASHREC));
         htmp->word = (char *) malloc(strlen(w) + 1);
         strcpy(htmp->word, w);
-        htmp->id = id;
+        htmp->num = id;
```

Comment:
I guess at a high level this should probably be moved to the common file as well...  I can do that though

Reply:
Do you mean _hashinsert()_? they have the same name, but it seems that they are a little bit different implemented.

Reply:
I meant hashsearch could be a general utility method.  But for now this refactoring is already quite useful, thank you